tests/unit/routes/reminders-test.js

```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
```

Comment:
If you have a test file lets make sure there's a legit assertion not just the boilerplate code.